tests/phpunit/tests/block-templates/WpBlockTemplatesRegistry.php

```diff
@@ -0,0 +1,270 @@
+<?php
+/**
+ * Test WP_Block_Templates_Registry class.
+ *
+ * @covers WP_Block_Templates_Registry
```

Comment:
https://docs.phpunit.de/en/10.5/annotations.html#coversdefaultclass

```suggestion
 * @coversDefaultClass WP_Block_Templates_Registry
```